README.md

```diff
@@ -1,118 +1,99 @@
-# Swanky Node
+# Swanky Node :sunglasses:
 
-Swanky node is the Substrate based blockchain configured to enable `pallet-contracts` and more features to help Wasm Smart Contract development.
+Swanky node is a Substrate based blockchain configured to enable `pallet-contracts` (a smart contract module) and more features to help WASM smart contract development locally.
 
 ## Features
-- pallet-contracts (polkadot-0.9.19) and its unstable-feature are enabled by default. ink version `3.0.0` is supported.
+- pallet-contracts (polkadot-0.9.19) and its unstable-feature are enabled by default.
 - `grandpa` & `aura` consensus were removed. Instead, `instant-seal` & `manual-seal` are used.
-- dApps staking
+  Blocks are authored (1) as soon as a transaction get in the pool (2) when `engine_createBlock` RPC called.
+  Blocks are finalized when `engine_finalizeBlock` RPC called.
+- dApps staking and ChainExtension to interact with it.
```

Comment:
dApp staking pallet you mean ? + maybe add a link to the Astar Frame repo of the pallet